.evergreen/init.sh

```diff
@@ -190,3 +190,44 @@ _init_sh_evg_dir="$(dirname "${_init_sh_this_file}")"
 # this one, and will therefore remain as native paths
 EVG_DIR="$(native_path "${_init_sh_evg_dir}")"
 LIBMONGOCRYPT_DIR="$(dirname "${EVG_DIR}")"
+
+# Executes CMake via the cache-managing script
+run_cmake() {
+    command bash "$EVG_DIR/cmake.sh" "$@"
+}
+
+# Executes CTest via the cache-managing script
+run_ctest() {
+    command bash "$EVG_DIR/ctest.sh" "$@"
+}
+
+EXE_SUFFIX=""
+if test "$OS_NAME" = "windows"; then
+    EXE_SUFFIX=".exe"
+fi
+
+if test "${USER_CACHES_DIR:=${XDG_CACHE_HOME:-}}" = ""; then
+    case "$OS_NAME" in
+    linux)
+        USER_CACHES_DIR=$HOME/.cache
+        ;;
+    macos)
+        USER_CACHES_DIR=$HOME/Library/Caches
+        ;;
+    windows)
+        USER_CACHES_DIR=${LOCALAPPDATA:-$USERPROFILE/.cache}
+        ;;
+    *)
+        log "Using ~/.cache as fallback user caching directory"
+        USER_CACHES_DIR="$(abspath ~/.cache)"
+    esac
+fi
+
+# Ensure we are dealing with a complete path
+USER_CACHES_DIR="$(abspath "$USER_CACHES_DIR")"
+
+: "${BUILD_CACHE_BUST:=1}"
+: "${BUILD_CACHE_DIR:="$USER_CACHES_DIR/libmongocrypt/build.$BUILD_CACHE_BUST"}"
+
+# Silence shellcheck:
+: "$LIBMONGOCRYPT_DIR,$EXE_SUFFIX"
```

Comment:
```suggestion
export LIBMONGOCRYPT_DIR
export EXE_SUFFIX
```

Recommend explicitly exporting variables that are intended to be used by the script invoker.

Reply:
This also exports to subprocesses, which isn't necessarily an issue, but something I wanted to avoid.